File: 101-200/161-170/168-excelColumnTitle/excelColumnTitle.py
class Solution(object):
    def convertToTitle(self, n):
        """
        :type n: int
        :rtype: str
        """

        result = ""

        while n:
            n -= 1
            result += chr(n % 26 + 65)
            n /= 26

        return result[::-1]

File: 101-200/161-170/168-excelColumnTitle/excelColumnTitle.cpp
#include <string>

using namespace std;

class Solution {
public:
    string convertToTitle(int columnNumber) {
        string result = "";
        for (; columnNumber-- > 0; columnNumber /= 26)
            result += 'A' + columnNumber % 26;

        reverse(result.begin(), result.end());
        return result;
    }
};